examples/text_summarization/bertsum_cnndm.ipynb

```diff
@@ -0,0 +1,3921 @@
+{
```

Comment:
<p>Would prefer consolidating all the imports in one cell. Some modules are being imported multiple times, e.g.</p><p>from utils_nlp.models.bert.extractive_text_summarization import Bunch</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp/pull/447/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>